src/shared/main/cpp/native_glue.cc

```diff
@@ -162,6 +162,8 @@ namespace litecore {
         // ATTN: In critical, should not call any other JNI methods.
         // http://docs.oracle.com/javase/6/docs/technotes/guides/jni/spec/functions.html
         jbyteArraySlice::jbyteArraySlice(JNIEnv *env, jbyteArray jbytes, bool critical)
```

Comment:
Can I do this with a default = 0 for length?

Reply:
Yes